src/core/response-function.ts

```diff
@@ -13,7 +13,10 @@ export class ResponseFN {
     const content = data;
 
     try {
-      this.data = JSON.parse(this.data).donneesSec;
+      const response = JSON.parse(this.data);
```

Comment:
Just realized that I didn't do `JSON.parse(content)` directly, can you also change this please ?